assets/scripts/menus/ShareMenu.jsx

```diff
@@ -123,12 +129,22 @@ export class ShareMenu extends React.Component {
       '&name=' + encodeURIComponent(getPageTitle(this.props.street)) +
       '&description=' + encodeURIComponent(shareText)
 
+    const signInLink = <a onClick={this.onClickSignIn} href="#">
+      <FormattedMessage
+        defaultMessage="Sign In with Twitter"
+        id="menu.share.sign-in"
```

Comment:
@flickz Would you remove the 'with Twitter' part?

@treyhahn @louh  How should we handle the translations since we're changing things around? I'd assume we'd want to make a new key and not mess w/ the existing translations?

Reply:
We'd definitely need a new key for the "Sign in" portion here.

Reply:
Thinking about this some more.

1. On the translation file, we should make a new key `menu.share.sign-in-twitter-link` that's just the translated part formerly inside the `<a></a>` tags. This would allow us to split the existing string into two.
2. Next, make a new key `menu.share.sign-in-link` that's just "Sign in" (without "with Twitter"). We could reserve how to translate these for later.

Reply:
@ryross @flickz Sorry for the very delayed reply on this! It looks like Lou already addressed it. To give you a quick view into the Transifex side of things:

When the name of a key is changed, Transifex assumes the old one has been deleted and a newly-named one has been added. So, please give me a heads up before merging a PR that changes a key name so that I can save any metadata and a current translation we have on the string.

If the key name stays the same but the source string (English version) changes, Transifex will treat it as the same key. In this case we can access the history of the key (see screenshot below) and the metadata will remain on the key, so we wouldn't lose anything.

![image](https://user-images.githubusercontent.com/12552044/41057555-65bc2700-6995-11e8-839e-805cda5ee8be.png)